pom.xml

```diff
@@ -114,7 +114,7 @@
         <repository>
           <id>sarxos</id>
           <name>sarxos</name>
-          <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+          <url>http://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
You don't want to revert this - also was this generated.. if it was that's great, that's how we manage dependencies.  Since I refreshed your branch, I hope a regen of the pom will the https, cuz it was put into the template

Reply:
Yeah that was generated, should be fixed with a regen

Reply:
@supertick Regen has fixed that regression, but it did also reorganize some of the dependencies due to my recent change to the generation code. Say, for example, we had service 1 and service 2 that both had the Apache commons dependency, before service 2 had the dependency in the pom.xml and service 1 had a comment that it was a duplicate. Now, some of them are reversed, so service 1 has the dependency, and service 2 has the comment about a duplicate. So functionally no difference, but the diff is very big and scary-looking

Reply:
:+1: